### PR TITLE
resource_manager: add is_background check

### PR DESCRIPTION
**Pull request by HuSharp**
add is_background check for tikv upload ru

### CODE REVIEW

File: proto/resource_manager.proto [hunk 1]
```diff
@@ -61,6 +61,8 @@ message TokenBucketsRequest {
   repeated TokenBucketRequest requests = 1;
   uint64 target_request_period_ms = 2;
   uint64 client_unique_id = 3;
+  // label background request.
+bool is_background = 4;
```
Comment (nolouch) [inline proto/resource_manager.proto:65]: format.

Comment (HuSharp) [reply proto/resource_manager.proto]: fixed.

**Review by nolouch — APPROVED** [review]

File: proto/resource_manager.proto [hunk 2]
```diff
@@ -61,6 +61,8 @@ message TokenBucketsRequest {
   repeated TokenBucketRequest requests = 1;
   uint64 target_request_period_ms = 2;
   uint64 client_unique_id = 3;
+  // label background request.
+  bool is_background = 4;
```
Comment (glorv) [inline proto/resource_manager.proto:65]: Is it better to put this field into TokenBucketRequest? We may want to batch both background/foreground request together

Comment (nolouch) [reply proto/resource_manager.proto:65]: cc @HuSharp  :)

Comment (HuSharp) [reply proto/resource_manager.proto:65]: Okay, I opened a new pr to change it.https://github.com/pingcap/kvproto/pull/1161

**Pull request merged**